addon/components/base/bs-tooltip.js

```diff
@@ -10,16 +10,18 @@ import layout from 'ember-bootstrap/templates/components/bs-tooltip';
 
  ```hbs
  <button class="btn">
-   {{bs-tooltip title="This is a tooltip"}}
+  <BsTooltip @title="This is a toolip" />
```

Comment:
Space before `/>` should be removed.

Reply:
According to this suggestion, we should be adding space?
https://github.com/ember-learn/guides-source/pull/614#discussion_r266114519

Reply:
Fine with that, let's just be consistent!